go-controller/pkg/metrics/master.go

```diff
@@ -38,7 +40,7 @@ var metricPodCreationLatency = prometheus.NewHistogram(prometheus.HistogramOpts{
 	Buckets:   prometheus.ExponentialBuckets(.1, 2, 15),
 })
 
-// metricPodCreationLatency is the time between a pod being scheduled and the
+// metricOvnCliLatency is the time between a pod being scheduled and the
```

Comment:
we should probably change this variable name later. We dont really use the cli anymore.